src/transforms/noop.rs

```diff
@@ -0,0 +1,33 @@
+use crate::config::{DataType, TransformConfig, TransformContext};
+use crate::event::Event;
+use crate::transforms::{FunctionTransform, Transform};
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+pub struct Noop;
+
+#[async_trait::async_trait]
+#[typetag::serde(name = "noop")]
+impl TransformConfig for Noop {
```

Comment:
Is this intentionally not added to `inventory` so that users can't configure it manually? That seems right to me but just wanted to double check.

Reply:
Yeah, that way I also don't have to document it 😅